[2022-05-19 15:59:00,164] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-18T15:55:44.156802+00:00 [queued]>
[2022-05-19 15:59:00,177] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-18T15:55:44.156802+00:00 [queued]>
[2022-05-19 15:59:00,178] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 15:59:00,178] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2022-05-19 15:59:00,179] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 15:59:00,190] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_docs> on 2022-05-18 15:55:44.156802+00:00
[2022-05-19 15:59:00,199] {standard_task_runner.py:52} INFO - Started process 945 to run task
[2022-05-19 15:59:00,203] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_great_expectations_dag', 'dbt_docs', 'scheduled__2022-05-18T15:55:44.156802+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_great_expectations_dag.py', '--cfg-path', '/tmp/tmpucvsb1nd', '--error-file', '/tmp/tmp5buohmhq']
[2022-05-19 15:59:00,204] {standard_task_runner.py:77} INFO - Job 152: Subtask dbt_docs
[2022-05-19 15:59:00,248] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-18T15:55:44.156802+00:00 [running]> on host 34ae02bc0b96
[2022-05-19 15:59:00,300] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=luciano.naveiro
AIRFLOW_CTX_DAG_ID=model_level_dbt_great_expectations_dag
AIRFLOW_CTX_TASK_ID=dbt_docs
AIRFLOW_CTX_EXECUTION_DATE=2022-05-18T15:55:44.156802+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-18T15:55:44.156802+00:00
[2022-05-19 15:59:00,302] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-19 15:59:00,303] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n        dbt docs generate --profiles-dir /opt/dbt/jaffle_shop --project-dir /opt/dbt/jaffle_shop\n        ']
[2022-05-19 15:59:00,323] {subprocess.py:85} INFO - Output:
[2022-05-19 15:59:04,250] {subprocess.py:89} INFO - 15:59:04  Running with dbt=1.1.0
[2022-05-19 15:59:04,297] {subprocess.py:89} INFO - 15:59:04  Unable to do partial parsing because config vars, config profile, or config target have changed
[2022-05-19 15:59:05,989] {subprocess.py:89} INFO - 15:59:05  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-19 15:59:05,995] {subprocess.py:89} INFO - 15:59:05
[2022-05-19 15:59:06,103] {subprocess.py:89} INFO - 15:59:06  Concurrency: 1 threads (target='dev')
[2022-05-19 15:59:06,105] {subprocess.py:89} INFO - 15:59:06
[2022-05-19 15:59:06,641] {subprocess.py:89} INFO - 15:59:06  Done.
[2022-05-19 15:59:06,743] {subprocess.py:89} INFO - 15:59:06  Building catalog
[2022-05-19 15:59:06,844] {subprocess.py:89} INFO - 15:59:06  Catalog written to /opt/dbt/jaffle_shop/target/catalog.json
[2022-05-19 15:59:08,224] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-19 15:59:08,274] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_great_expectations_dag, task_id=dbt_docs, execution_date=20220518T155544, start_date=20220519T155900, end_date=20220519T155908
[2022-05-19 15:59:08,395] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-19 15:59:08,466] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
